anvil.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -e
+
+PORT=8545
+HOST="0.0.0.0"
+SLEEP_DURATION=0.1
+
+# Start the Anvil server.
+anvil --host "$HOST" --rpc-url "$MAINNET_RPC_URL" &
+ANVIL_PID=$!
+
+# Wait until Anvil is ready
+while ! nc -z localhost "$PORT"; do sleep "$SLEEP_DURATION"; done
+
+# Enable auto-impersonation
+cast rpc anvil_autoImpersonateAccount true
+
+# Deploy contracts, forking from the provided RPC URL
+forge script script/Deploy.s.sol --rpc-url "http://localhost:$PORT" --unlocked --broadcast --sender "$DEPLOYER"
```

Comment:
`DEPLOYER` here is the EOA address? What' the mnemonic that's used by default? Would be nice to be able to customize that or pass in an address that ownership will be transferred to (for things like setting the trusted caller or changing storage price)

Reply:
Yeah, `DEPLOYER` is the EOA that will send the deployment transactions. This needs to be set as the owner of `IdRegistry` and `KeyRegistry`, and the role admin of `StorageRegistry` in order to perform the post-deploy setup. steps.

The deploy scripts assume we have the values in [`.env`](https://github.com/farcasterxyz/contracts/blob/a7321bd06bf5caae6185de9457cd10a458e0dcaf/.env.example) in the environment. I'll add some more documentation on what all these values are and maybe even reasonable defaults in ` .env.example`.